Dockerfile

```diff
@@ -3,7 +3,15 @@ WORKDIR /app
 RUN apk add npm
 COPY ./package.json /app
 COPY ./package-lock.json /app
-RUN npm i --no-dev
+# templates are needed in order to compile css
+COPY ./tmpl /app/tmpl
+# unsafe-perm makes sure we can write css to node_modules
+RUN npm i --no-dev --unsafe-perm
```

Comment:
So this flag seems like a workaround to us using root user to create all this things: https://geedew.com/What-does-unsafe-perm-in-npm-actually-do/

We should rather make sure we use the correct user (www-data) instead of adding this flag. But that might be something for a different PR. In any case the comment explaining the flag is only half right. At the end of the day, using this flag ideally should be avoided and we should do this properly (yes, I know, I was lazy when I set this up too and now look at who is complaining)